pandas/tests/test_multilevel.py

```diff
@@ -242,12 +244,11 @@ def test_std_var_pass_ddof(self):
             expected = df.groupby(level=0).agg(alt)
             tm.assert_frame_equal(result, expected)
 
-    @pytest.mark.parametrize("klass", [Series, DataFrame])
```

Comment:
we ought to make some rules to prohibit this in favor of the fixture (maybe not be perfect but can prob catch most of these)